### PR TITLE
Include Salesforce Sans

### PR DESCRIPTION
**Pull request by mattrothenberg**
Includes the `Salesforce Sans` font-family so that our UI doesn't show up in Times New Roman 😛 .

Hitherto, my browser had been loading a system copy of Salesforce Sans, which meant that I wasn't able to observe this issue.

### CODE REVIEW

File: .storybook/manager-head.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<style>
```
Comment (uhoh-itsmaciek) [inline .storybook/manager-head.html]: `manager-head.html` is for the Storybook UI itself--do we want Salesforce Sans there too? We haven't really attempted to customize it at all afaik...

Comment (mattrothenberg) [reply .storybook/manager-head.html]: @uhoh-itsmaciek Yes, otherwise the navigation items will appear in `Times New Roman`

Comment (mattrothenberg) [reply .storybook/manager-head.html]: We're lightly customizing the appearance via: https://github.com/heroku/react-hk-components/blob/master/.storybook/heroku-theme.js

Comment (uhoh-itsmaciek) [reply .storybook/manager-head.html]: Ah cool, carry on.

**Pull request merged**